src/app/friends-list/friends-list.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Observable, Subscription } from 'rxjs';
+import { debounceTime, distinctUntilChanged } from 'rxjs/operators';
+import { Friend, FriendRequest } from '../models/user.model';
+import { UserService } from '../user.service';
+
+@Component({
+  selector: 'app-friends-list',
+  templateUrl: './friends-list.component.html',
+  styleUrls: ['./friends-list.component.scss']
+})
+export class FriendsListComponent implements OnInit, OnDestroy {
+  //******************* FIELD DECLARATIONS
+  usersFriends$: Observable<Friend[] | undefined>;
```

Comment:
it`s not so good idea to set `| undefined` everywhere, better to use only Interfase